examples/Sql/CassandraDataFrame/Program.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Spark.CSharp.Core;
+using Microsoft.Spark.CSharp.Sql;
+
+namespace Microsoft.Spark.CSharp.Examples
+{
+    /// <summary>
+    /// This example shows how to use Cassandra in .NET Apache Spark DataFrame API
+    /// It covers load rows from Cassandra, filter it and save results to another table
+    /// </summary>
+    public class CassandraDataFrameExample
+    {
+        static void Main(string[] args)
+        {
+            var cassandraHostName = "localhost";
```

Comment:
It will be nice to the option of passing cassandra hostname/keyspace/table through arguments (or app.config).